_posts/2025-08-15-freedom_fuel.md

```diff
@@ -0,0 +1,31 @@
+
+While that claim might not be true, this "pact with the devil", as Kurzgesagt calls it, might actually have had a much more *technical* benefit for early societies. Ethanol is quite effective as a crude food preservative &mdash; and what's even better, alcohol fermentation pretty much happens on its own. It was an invaluable tool for our earliest ancestors, who oftentimes had to endure long periods without access to fresh fruits or grains. I'm far from advocating **for** ingesting alcohol, as nowadays we have healthier and more effective ways of stopping our fruits from going bad &mdash; but it might not be worth pouring it all down the drain. We may have not yet seen the good old *aqua vitae*'s full potential.
+
+### What makes my engine tick
+Biofuels are nothing new. Historically cars used to run on all kinds of combustible liquids. In fact pure ethanol powered Henry Ford's first car, the Quadricycle &mdash; it only fell out of favor during the Prohibition due to gasoline being cheaper[^gasoline]. Contemporary gasoline all over the world usually contains at least some ethanol. Brazil went even further &mdash; all new vehicles sold there are flexfuel-capable, meaning they can run on a blend of gasoline and ethanol in pretty much any proportion. It makes sense &mdash; it lets Brazil shield itself from oil shortages and cut CO₂ emissions. In fact ethanol is superior to gasoline in many regards: it is more resistant to knock, allowing running higher compression ratios for higher efficiency and more torque; its air-fuel ratio is better, allowing extracting more power from a given cylinder volume; it burns cleaner and has much lower carbon footprint[^ecology]; it makes the engine run cooler[^cool]; and finally it's fully renewable. The only issue is its energy density, which is around 67% of gasoline, meaning the engine has to burn more fuel to produce the same power. It's a no-brainer[^caveat].
+
+### What fuels the cells
+Combustion, however cozy and heartwarming, is not the only way of extracting energy back out of alcohol (or rather alcohols in general). Fuel cells can convert alcohol into electricity &mdash either directly[^DEFC] or indirectly &mdash; and do it easily and efficiently. They may not outperform hydrogen fuel cells, but since hydrogen is notoriously difficult to store, the lower cost of infrastructure for liquid fuels is a huge advantage. Even more importantly, it is possible to produce alcohols electrochemically[^methanol][^ethanol] &mdash; which opens another possibility of storing energy from renewable sources, further smoothing out the fluctuations in available power.
```

Comment:
Again, might be worth explaining why this technology is relatively obscure despite the obvious benefits 🤔

Reply:
Reworded to highlight that it's currently being researched.